README.md

```diff
@@ -1,9 +1,24 @@
+
 # Operators for Qiskit
 
-###  - [Qiskit developer's exp operator](https://github.com/husky-parul/glowing-quantum/tree/qiskit-dev-op/operators-examples/qiskit-dev-operator)
+## Overwiew
+
+These operators integrates your quantum workloads on OpenShift. It provides a development environment to implement quantum algorithms and runs them either on IBM quantum computers or simulators.
```

Comment:
If operators is meant to be plural then the use "integrate", else make operators singular.

Reply:
I still think _integrates_ should be changed to _integrate_